consensus/types/round_state_test.go

```diff
@@ -2,12 +2,12 @@ package types
 
 import (
 	"testing"
-	"time"
 
-	amino "github.com/tendermint/go-amino"
```

Comment:
please don't remove this alias

Reply:
IDE removed it. Goland is great but not very flexible when it comes to configuration.